### ISSUE TITLE
about model.load with weights_only

### ISSUE THREAD

**Issue by wanboyang** [tflearn/tflearn#655]
my tflearn edtion:0.3.0
tensorflow edtion:1.0.0

when I load my model with weights_only=True,my model looks like have not been trained.The accuracy is low.
```
def mynet(input):
    CONV1 = conv_2d(input, 96, 7, strides=2, activation=None,name='CONV1_1')
    BN1 = batch_normalization(CONV1,name='BN1')
    AC1 = relu(BN1)
    x = max_pool_2d(AC1, 3, strides=2,name='pool1')
    x = conv_2d(x, 256, 5, strides=2,activation=None,name='CONV2_1')
    x = batch_normalization(x,name='BN2')
    x = relu(x)
    x = max_pool_2d(x, 3, strides=2,name='pool2')
    x = conv_2d(x, 384, 3, activation=None,name='CONV3')
    x = batch_normalization(x, name='BN3')
    x = relu(x)
    x = conv_2d(x, 384, 3, activation=None,name='CONV4')
    x = batch_normalization(x, name='BN4')
    x = relu(x)
    x = conv_2d(x, 256, 3, activation=None,name='CONV5')
    x = batch_normalization(x, name='BN5')
    x = relu(x)
    x = max_pool_2d(x, 3, strides=2,name='pool3')
    x = fully_connected(x, 4096, activation='relu',name='FC1')
    x = fully_connected(x, 4096, activation='relu',name='FC2')
    x = fully_connected(x, 2, activation='softmax',name='OUT')
    return  x

x = input_data(shape=[None, 224, 224, 3], name='input')
pre=mynet(x)
network = regression(pre, optimizer="momentum",
                     loss='categorical_crossentropy',metric='accuracy')
model = tflearn.DNN(network)
model.load(model_file='./new_test1_finetune_model9761',weights_only=True)
accuracy_test1 = model.evaluate(X_test1, Y_test1, batch_size=64)
"
```
accuracy_test1 is only 0.5
but when I use
`model.load(model_file='./new_test1_finetune_model9761')`

accuracy_test1 is  0.976.

I can not change my metrics function if don't use weights_only=True.
if I do 
`network = regression(pre, optimizer="momentum",
                     loss='categorical_crossentropy',metric='top1')
model = tflearn.DNN(network)
model.load(model_file='./new_test1_finetune_model9761')`
I will see
`	 [[Node: save_1/RestoreV2_64 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save_1/Const_0, save_1/RestoreV2_64/tensor_names, save_1/RestoreV2_64/shape_and_slices)]]
W tensorflow/core/framework/op_kernel.cc:993] Not found: Key StandardError/truediv/moving_avg not found in checkpoint`

so what should I do if I want change my metric function?

**Comment by SadeghMSalehi** [comment]
I have the same problem. It seems the problem is this: when you load your model with weights_only=True, the Batch Normalization weights are not loaded. That's why your whole model is not working! I'm working to solve this problem. Please let me know if you find any solution.

**Comment by SadeghMSalehi** [comment]
I solved the problem by using "local_response_normalization" instead of "Batch Normalization"
